app/assets/stylesheets/config/_colors.css

```diff
@@ -293,23 +239,37 @@ of particular colors in components and views.
 
   /* CTAs: Default */
   --cta-bg: transparent;
-  --cta-bg-hover: var(--accent-brand-a10);
-  --cta-color: var(--base-80);
+  --cta-bg-hover: rgba(var(--accent-brand-rgb), 0.1);
+  --cta-color: rgb(var(--grey-800));
   --cta-color-hover: var(--accent-brand-darker);
-  --cta-border: var(--base-60);
+  --cta-border: rgb(var(--grey-600));
   --cta-border-hover: var(--accent-brand-darker);
 
   /* CTAs: Branded */
   --cta-branded-bg: transparent;
   --cta-branded-bg-hover: var(--accent-brand);
   --cta-branded-color: var(--accent-brand);
-  --cta-branded-color-hover: var(--base-inverted);
+  --cta-branded-color-hover: rgb(var(--white));
   --cta-branded-border: var(--accent-brand);
   --cta-branded-border-hover: var(--accent-brand-darker);
 
+  /***********************************************
+  ** Tooltips ************************************
+  ***********************************************/
+
+  --tooltip-bg: rgba(var(--grey-900), 0.9);
+  --tooltip-color: rgb(var(--white));
+
   /***********************************************
   ** Utilities ***********************************
   ***********************************************/
 
+  /* Focus */
   --focus: var(--accent-brand);
+
+  /* Shadows */
+  --shadow-1: 0 10px 15px -3px rgba(var(--black), 0.1),
+    0 4px 6px -2px rgba(var(--black), 0.05), 0 0 0 1px rgba(var(--black), 0.1);
+  --shadow-2: 0 20px 25px -5px rgba(var(--black), 0.1),
+    0 10px 10px -5px rgba(var(--black), 0.05), 0 0 0 1px rgba(var(--black), 0.1);
```

Comment:
These should TECHNICALLY be in "variables.css" instead of theme file BUT we actually tweak shadows a bit for dark theme so it made sense to move them here.